en/docs/reference/pricing.md

```diff
@@ -2,229 +2,145 @@
 
-<br/>
+| Free           |  Pay As You Go  | Enterprise |
+|----------------|-----------------|------------|
+| In-built monitoring dashboard<br/><br/>Logs<br/>(7 days of data retention)<br/><br/>Traces<br/>(7 days of data retention)<br/><br/>Alerting<br/>(7 days of data retention) | In-built monitoring dashboard<br/><br/>Logs<br/><br/>Traces<br/><br/>Alerting<br/><br/>Extending to 3rd Parties | In-built monitoring dashboard<br/><br/>Logs<br/><br/>Traces<br/><br/>Alerting<br/><br/>Extending to 3rd Parties |
 
-#### Update jet fuel price daily
+#### Security
 
-![ Update jet fuel prices daily](../assets/img/references/scheduled-task-example1.png)
+Choreo supports the following security features for all the plans:
 
-Let’s see how we can calculate the steps for the update maintenance process:
+- SSO for API Developer Portal
+- Enterprise Login
+- Container manifest scanning
+- Container static scanning
+- Run-time container security
 
-- The scheduled task ([scheduler](#scheduler)) on Choreo runs once per day.
-- The scheduler makes **1 API call to the [Oil Price API](#oilPriceAPI)** to  retrieve the daily oil price.  This is counted as **1 step**. 
-- Upon receiving the daily oil price, the Scheduler then applies the formula to calculate the jet fuel price and calculates the daily jet fuel price.
-- The Scheduler on Choreo then makes **1 database call to the [Air Fare Database](#airFareDatabase)** to update the daily jet fuel price. This is counted as **1 step**. 
+#### Cloud-code editing
 
+Choreo supports cloud-code editing for the three plans as explained below:
 
-- **Assuming the entire execution time for the update process takes 1200 ms**, the total number of steps can be calculated as follows:
-  
-        Number of API calls/database calls/events = 2 steps
-        Number of computation units = round((1200-500)/500) = 2 units = 2 steps
-        Total number of steps = 2 steps + 2 steps = 4 steps
+| Free           |  Pay As You Go  | Enterprise |
+|----------------|-----------------|------------|
+| 10 hours on the Choreo Web Editor. | 10 hours are allowed on the Choreo Web Editor per component free of charge.<br/><br/>If you need more code editing time, contact our Sales team to request a fully managed Cloud Coding Cluster. | If you have chosen the Enterprise plan, you may have a Cloud Coding Cluster already deployed on your infrastructure.<br/><br/>If you do not already have a Cloud Coding Cluster and you need one, contact our Sales team. |
 
-<br/>
+### Support
 
-#### Update airfare daily  
+Community support is available for all three plans. In addition, paid support is available for Pay As You Go and Enterprise plans.
 
-![ Update airfare daily](../assets/img/references/scheduled-task-example2.png)
+| Free           |  Pay As You Go  | Enterprise |
+|----------------|-----------------|------------|
+| Community support | Community support and paid plans | Community support and paid plans|
```

Comment:
How can someone get info for the paid plans for support? Shall we link them?

Reply:
Linked the Billing Portal.